app/components/helpscout-beacon.js

```
@@ -0,0 +1,45 @@
+    try {
+      if (typeof window.Beacon === 'function') {
+        window.Beacon('init', 'bb089ae9-a4ae-4114-8f7a-b660f6310158');
+        console.log('HelpScout Beacon initialized.');
+        this.beaconInitialized = true;
+      } else {
+        console.warn('HelpScout Beacon not available after script load.  Check network request.');
+      }
+    } catch (error) {
+      console.error('Error loading HelpScout Beacon:', error);
+    }
+  }
```

ESLint

[error] Member initBeacon should be declared before member willDestroy.

(@typescript-eslint/member-ordering)